### DISCUSSION TITLE
how do I allow data:image/ in img tag?

### DISCUSSION THREAD

**Discussion by Xyncgas** [mganss/HtmlSanitizer#337] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/35312807/154869063-3d16dca9-30f3-4246-a06f-6691ed8c4dbc.png)
I am pretty sure this gets sanitised by default

**Comment by mganss** [comment, marked as answer]
Call `sanitizer.AllowedSchemes.Add("data");`

**Reply by Xyncgas** [reply]
Thanks

**Comment by tiesont** [comment]
Seems worth mentioning that this specific question is answered in the wiki: https://github.com/mganss/HtmlSanitizer/wiki/Examples#ex2-allowing-data-uris

Not saying it's wrong to ask, but it's always a good idea to check the documentation and examples.

**Comment by isaacbromberg** [comment]
Wouldn't it make sense to make it more precise so that the allowed schema would be:
sanitizer.AllowedSchemes.Add("data:image");

We have got security advice to allow only images in our form.

**Reply by mganss** [reply]
The scheme is `data`, `image/png` is the MIME type of the data. If you want more control over the allowed URLs, you can use the `FilterUrl` event.
